src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -324,6 +320,53 @@ public class InstancesController : ControllerBase
             }
         }
 
+        /// <summary>
+        /// Allows an app owner to update the substatus of an instance.
+        /// </summary>
+        /// <param name="org">unique identifier of the organisation responsible for the app</param>
+        /// <param name="app">application identifier which is unique within an organisation</param>
+        /// <param name="instanceOwnerPartyId">The party id of the instance owner.</param>
+        /// <param name="instanceGuid">The id of the instance to confirm as complete.</param>
+        /// <param name="substatus">The new substatus of the instance.</param>
+        /// <returns>Returns the instance with updated list of confirmations.</returns>
+        [Authorize]
```

Comment:
This work for now, but we probably need to figure out how we can set this is xacml in case  a org wants other org to set status, but this is not somthing that has been requested

Reply:
I agree